### PR TITLE
luci-app-pbr: sync with principal package

### PR DESCRIPTION
**Pull request by stangri**
Depends on https://github.com/openwrt/packages/pull/24920

One update I'd like some feedback for is changing include of `luci.mk` from relative to an absolute path, that was requested by some users of upstream repo. If it's going to be a problem in OpenWrt repo, please let me know.

### CODE REVIEW

File: applications/luci-app-pbr/Makefile [hunk 1]
```diff
@@ -24,6 +24,6 @@ help
 	Version: $(PKG_VERSION)-$(PKG_RELEASE)
 endef
 
-include ../../luci.mk
+include $(TOPDIR)/feeds/luci/luci.mk
```
Comment (krant) [inline applications/luci-app-pbr/Makefile:27]: @stangri this change causing that in my local setup (and I'm not even building it):
```
ERROR: please fix package/feeds/applications/luci-app-pbr/Makefile - see logs/package/feeds/applications/luci-app-pbr/dump.txt for details
```
My luci.mk lives in `openwrt/package/feeds/luci.mk` and for all other packages it causes no problem.

Comment (stangri) [reply applications/luci-app-pbr/Makefile:27]: Thanks for the feedback, could you please provide the content of the `logs/package/feeds/applications/luci-app-pbr/dump.txt`?

Comment (krant) [reply applications/luci-app-pbr/Makefile:27]: Here you are:
```
Makefile:27: /home/krant/openwrt/feeds/luci/luci.mk: No such file or directory
make[2]: *** No rule to make target '/home/krant/openwrt/feeds/luci/luci.mk'.  Stop.
```

Comment (stangri) [reply applications/luci-app-pbr/Makefile:27]: @krant thanks for your prompt reply, I'll revert the change

**Pull request merged**